Install on Atom 124 fails

During the install of geojson--preview, I get:

npm ERR! runas@0.5.4 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the runas@0.5.4 install script.
npm ERR! This is most likely a problem with the runas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls runas
npm ERR! There is likely additional logging output above.

The whole error printout is here:
https://github.com/backspaces/play/blob/master/geojsonAtomError.txt

Let me know if this should go to a different repo's issues.  I know this release of Atom had many deep changes, so the issue may be theirs!

   -- Owen

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "mapbox.js": "1.6.x",
     "geojsonhint": "0.2.x",
     "fs-plus": "2.x",
-    "pathwatcher": "^1.0",
+    "pathwatcher": "^2.0.10",
     "underscore-plus": "1.x"
   }
 }

